File: WordGames/QnoU/QnoU.py
# Program to create a list for all of the words containing a Q but not a U?

# importing the word list
sow_pods = open("/home/evergreen/UD/.idea/SowPods")

# Go through each word in the file and add it to the list, words
words = []
for line in sow_pods:
    words.append(line.strip())

# Creating variables
QnoU = []

for w in words:
    if ("Q" in w) and ("U" not in w):
        QnoU.append(w)

print("This is all of the words containing Qs and no Us:\n" + str(QnoU))

File: WordGames/QnoU/qNoU.js
// Program to find all the words with Q's but no U's
// Importing the fs module
let fs = require("fs")

// Intitializing the readFileLines with the file
const readFileLines = filename =>
   fs.readFileSync(filename)
   .toString('UTF8')
   .split('\n');

// Calling the readFiles function with file name
let words = readFileLines('/home/evergreen/UnderDogsAssignments/sowpods');
console.log(typeof(words))
qNoUs = []

for(let i = 0; i < words.length; i++) { 
    if (words[i].includes("Q") && !words[i].includes("U")) {
        qNoUs.push(words[i]);
    }
}

console.log("The words that contains Q's but not U's are " + qNoUs)